index.html

```diff
@@ -49,5 +49,148 @@ <h2>I'm a Software Developer</h2>
+                </div>
+            </div>
+            <div class="genericCard1">
+                <div class="cardContent2">
+                    <h2>Profesional Art Printing Data</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                    <ul class="categories">
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="projectsButton2">See project</a>
+            </div>
+            <div class="genericCard2">
+                <div class="cardContent2">
+                    <h2>Data Dashboard Healthcare</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                    <ul class="categories">
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="projectsButton2">See project</a>
+            </div>
+            <div class="genericCard3">
+                <div class="cardContent2">
+                    <h2>Website Portfolio</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                    <ul class="categories">
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="projectsButton2">See project</a>
+            </div>
+            <div class="genericCard1">
+                <div class="cardContent2">
+                    <h2>Profesional Art Printing Data More</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                    <ul class="categories">
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="projectsButton2">See project</a>
+            </div>
+            <div class="genericCard2">
+                <div class="cardContent2">
+                    <h2>Data Dashboard Healthcare</h2>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                    <ul class="categories">
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <a class="projectsButton2">See project</a>
```

Comment:
- [ ] Please kindly implement the button hover, and pressed functionalities like displayed in the Figma design. please add for all your buttons. 

![Screenshot (1214)](https://github.com/JhonatanErosa/Portfolio/assets/65298048/45c66a5e-4a59-48ac-99db-ed5fd18f75cf)

Reply:
Excellent suggestion 👍